CV-HTML-CSS/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <script
+      src="https://kit.fontawesome.com/4d62024606.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>CV Lika Kharatishvili</title>
+  </head>
+  <body>
+    <div class="contact-wrapper">
+      <div class="contact-image">
+        <div class="info">
+          <span><i class="fas fa-map-marker-alt"></i>Address</span>
+          <p>Tbilisi, Georgia</p>
+        </div>
+        <div class="info">
+          <span><i class="fas fa-phone-alt"></i>Phone</span>
+          <p>(+955) 599 34 51 46</p>
+        </div>
+        <div class="info">
+          <span><i class="fas fa-envelope-open-text"></i>Email</span>
+          <p>lika.likaratishvili01@btu.edu.ge</p>
+        </div>
+        <div class="info">
+          <span><i class="fab fa-telegram-plane"></i>Telegram</span>
+          <p>@Likaxara</p>
+        </div>
+        <div class="info">
+          <span><i class="fab fa-skype"></i>Skype</span>
+          <p>live:.cid.5b4f87fbbd8ee139</p>
+        </div>
+      </div>
+      <div class="right-part">
+        <div class="summary">
+          <h2>Summary</h2>
+          <p>
+            For me, front-end development is a critical component of what I’m
+            passionate about, so I take it seriously. It enabled me to easily
+            put my ideas into digital form and get results back really quick.
+            Mostly I’m self taugh, trying really hard to become an awesome
+            front-end developer. I take a lot of Udemy and Pluralsight courses.
+            <strong
+              >I always going to do my best, no matter where I am.
+            </strong>
+          </p>
+        </div>
+        <div class="skills">
+          <h2>Skills</h2>
+          <ul>
+            <li>HTML5</li>
+            <li>CSS3</li>
+            <li>Javascript</li>
+            <li>Angular</li>
+            <li>Git</li>
+          </ul>
+        </div>
+        <div class="education">
+          <h2>Education</h2>
+          <div>
+            <h3><strong>General Education Diploma</strong></h3>
+            <p>Buckswood International School</p>
+            <p><i>2004 - 2016</i></p>
```

Comment:
change i to em, please -A screen reader will pronounce the words in <em> with an emphasis, using verbal stress.